ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -131,6 +131,19 @@
   @Default("true")
   public final boolean routerGetCrossDcEnabled;
 
+  /**
+   * Indicates whether get operations are allowed to make requests to nodes in non-originating remote data centers.
+   */
+  @Config("router.get.include.non.originating.dc.replicas")
+  @Default("true")
+  public final boolean routerGetIncludeNonOriginatingDcReplicas;
```

Comment:
Do you like this name? 
False is the value we prefer to improve read-after-write performance.